src/guide/making-a-change.md

```diff
@@ -0,0 +1,73 @@
+# Making a Change
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [CSR Wiki](https://wiki.openjdk.org/display/csr/Main)
+:::
+
+In case you jumped directly here and skipped reading the earlier sections of this guide, please keep in mind that there's a lot more to it than just making a change in the code and submit a PR to GitHub. The list below shows the bare minimum required before a change can be accepted into OpenJDK.
+
+#. **Discuss the intended change** -- See [Contributing to an OpenJDK Project]
+#. **Make sure an issue id exists for the work** -- See [Filing an issue]
+#. **Initiate a CSR request if your change have a compatibility impact** -- See [Working with the CSR]
+#. **Fix the issue** -- See [Cloning the JDK], [Working with git branches], and [Building the JDK]
+#. **Write regression tests and run relevant regression and unit tests on all relevant platforms** -- See [Testing the JDK]
+#. **Create a changeset** -- See [Working With Pull Requests]
+#. **Update the bug content** -- See [Updating an issue while fixing]
+#. **Request a review of the changes** -- See [Life of a PR]
+#. **Merge with the latest upstream changes and test again**
+#. **Integrate the changeset** -- See [Working With Pull Requests]
+#. **Write a release note if appropriate** -- See [Release Notes]
+
+## Working with the CSR
+
+Changes that have a compatibility impact will require a separate approval besides the code review. This is handled through a [Compatibility and Specification Review](https://wiki.openjdk.org/display/csr/Main), a.k.a. CSR. Compatibility impact can be things like requiring a specification change, directly affect an external interface, changing command line options, or otherwise alter the behavior of the JDK in ways that could cause issues for users when upgrading to the next JDK version.
+
+See the [CSR wiki](https://wiki.openjdk.org/display/csr/Main) for information on how to work through the CSR process.
+
+The CSR must be approved before the change is allowed to be integrated to a feature release or update release repository. Feature design and implementation work may begin concurrently with the CSR review, but may need to be modified in response to CSR feedback.
+
+## Copyright Headers
+
+All source code files in OpenJDK has a header with copyright statements and a license. Since this is legal documentation it shall not be updated or modified without seeking guidance from your legal representative. For that reason this guide can't really give detailed information on what you can or should do. There are however a few generic things that can be clarified.
```

Comment:
```suggestion
All source code files in OpenJDK have a header with copyright statements and a license. Since this is legal documentation it shall not be updated or modified without seeking guidance from your legal representative. For that reason this guide can't really give detailed information on what you can or should do. There are however a few generic things that can be clarified.
```